src/Atmos/Model/source.jl

```diff
@@ -182,6 +183,49 @@ function atmos_source!(
     end
 end
 
+"""
+    CreateClouds{FT} <: Source
+
+A source/sink to `q_liq` and `q_ice` implemented as a relaxation towards
+equilibrium in the Microphysics module.
+The default relaxation timescales are defined in CLIMAParameters.jl.
+"""
+struct CreateClouds <: Source end
+function atmos_source!(
+    ::CreateClouds,
+    atmos::AtmosModel,
+    source::Vars,
+    state::Vars,
+    diffusive::Vars,
+    aux::Vars,
+    t::Real,
+    direction,
+)
+    # current phase partition
+    FT = eltype(state)
+    ts = thermo_state(atmos, state, aux)
+    q = PhasePartition(ts)
+    T = air_temperature(ts)
+
+    # saturation specific humidity over liquid (packed into PhasePartition)
+    q_liq_sat =
+        q_vap_saturation_generic(atmos.param_set, T, state.ρ, phase = Liquid())
```

Comment:
Remind me to write a wrapper for this so that you can call this with
```suggestion
    q_liq_sat = q_vap_saturation(ts, Liquid())
```